packages/atomic/tsconfig.json

```diff
@@ -5,10 +5,10 @@
     "declaration": false,
     "experimentalDecorators": true,
     "lib": ["dom", "es2020", "ESNext.Collection"],
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
+    "module": "ES2022",
     "resolveJsonModule": true,
     "useDefineForClassFields": false,
-    "module": "esnext",
     "target": "es2019",
```

Comment:
To interact in typescript with a dependency that uses the new exports field, it is required to have moduleResolution set to either esnext (also named node16) or bundler. In our case,  we will most likely use Bundler for all our packages.